python_modules/dagster/dagster/_utils/caching_instance_queryer.py

```diff
@@ -56,7 +58,7 @@
 RECORD_BATCH_SIZE = 1000
 
 
-class CachingInstanceQueryer(DynamicPartitionsStore):
+class CachingInstanceQueryer(DynamicPartitionsStore, LoadingContext):
```

Comment:
connected to this discussion https://github.com/dagster-io/dagster/pull/24282#discussion_r1759575841

not sure the thinking from there applies here though - I feel like it would be better for this to get the loader from above instead of be its own. Maybe add a `for_test` constructor and `TestLoadingContext` for the test call sites.

Reply:
updated the upstream PR to remove this subclass